addons/godot-firebase/firestore/firestore_task.gd

```diff
@@ -157,11 +157,17 @@ func _on_request_completed(result : int, response_code : int, headers : PoolStri
     emit_signal("task_finished", self)
 
 func emit_error(signal_name : String, bod, task) -> void:
-    if bod and bod.size() > 0:
-        error = bod[0].error
+    if bod:
+        if bod is Array and bod.size() > 0:
```

Comment:
Do we not need to check if "bod.has("error")" here?

Reply:
No, but we _do_ need to check if bod[0].has("error"), darnit. Good call out!